.github/workflows/build.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     runs-on: ${{matrix.os.name}}
     strategy:
       matrix:
-        os: [{ name: ubuntu-22.04, bin-name: linux }] # { name: windows-2022, bin-name: windows }, { name: macos-12, bin-name: darwin }
-        arch: [amd64] # arm64
+        os: [{ name: ubuntu-22.04, bin-name: linux }, { name: macos-14, bin-name: darwin }] # { name: windows-2022, bin-name: windows }
```

Comment:
For neo-go we use macos-12 - https://github.com/nspcc-dev/neo-go/blob/master/.github/workflows/build.yml#L40

Reply:
fun fact, macos-12 runs on x86, macos-14 on arm, so I guess macos-14 is the correct one

Reply:
For builds it doesn't really matter, the resulting binary is determined by GOOS/GOARCH and you can build any binary on ubuntu-22.04 in fact (one of Go surprises).